# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1302091269 -7200
#      Wed Apr 06 14:01:09 2011 +0200
# Node ID 6442fb2e1dfa3c9218e611594c1b588159a1344d
# Parent  eb510dd290bfdd0054c9d7147129a47cbcd71a89
Readded info request at startup.

diff --git a/gui//octaveterminal.cpp b/gui//octaveterminal.cpp
--- a/gui//octaveterminal.cpp
+++ b/gui//octaveterminal.cpp
@@ -60,16 +60,17 @@ void OctaveTerminal::blockUserInput() {
 void OctaveTerminal::allowUserInput() {
     m_commandLine->setEnabled(true);
     m_commandLine->setFocus();
 }
 
 void OctaveTerminal::assignClient(Client *client) {
     m_client = client;
     allowUserInput();
+    addRequest("info\n");
 }
 
 void OctaveTerminal::showEnvironment() {
     addRequest("who\n");
 }
 
 void OctaveTerminal::handleAnsweredRequest() {
     allowUserInput();
